gulpfile.js

```diff
@@ -0,0 +1,24 @@
+const del = require('del');
+const eslint = require('gulp-eslint');
+const gulp = require('gulp');
+const zip = require('gulp-zip');
+
+gulp.task('build', ['lint', 'clean:dist'], function () {
+  return gulp.src('src/*')
+    .pipe(zip('addon.xpi'))
+    .pipe(gulp.dest('dist'));
+});
```

Comment:
Nothing too interesting. Basically this task runs the `lint` and `clean:dist` tasks and then creates a `dist/addon.xpi` file from the contents of `src/*` (including the src/install.rdf and src/update.rdf files).

We can do a future PR to copy the *.rdf files into dist/ too if that seems useful.